src/Phan/Language/Element/Method.php

```diff
@@ -166,19 +174,48 @@ public function isOverriddenByAnother(): bool
         return $this->getPhanFlagsHasState(Flags::IS_OVERRIDDEN_BY_ANOTHER);
     }
 
+    /**
+     * Returns a (possibly incomplete) list of methods overriding this one.
+     * @param CodeBase $code_base
+     * @return list<Method>
+     * @suppress PhanUnreferencedPublicMethod May be called by plugins (#4502)
+     */
+    public function getPossibleOverrides(CodeBase $code_base): array {
+        $ret = [];
+        foreach ($this->method_overrides as $fqsen) {
+            // Consistency checks, mostly for language server mode.
+            /** @var FullyQualifiedMethodName $fqsen */
+            if (!$code_base->hasMethodWithFQSEN($fqsen)) {
+                continue;
+            }
+            $method = $code_base->getMethodByFQSEN($fqsen);
+            $subclassExpanded = $method->getClass($code_base)->getFQSEN()->asType()->asExpandedTypes($code_base);
```

Comment:
It's more efficient not to call getClass here and below and less prone to throwing in the future

```suggestion
            $subclassExpanded = $method->getClassFQSEN()->asType()->asExpandedTypes($code_base);
```

```php
// src/Phan/Language/Element/ClassElement.php
    /**
     * @return FullyQualifiedClassName
     * The FQSEN of the class on which this element lives
     */
    public function getClassFQSEN(): FullyQualifiedClassName
    {
        return $this->class_fqsen;
    }
```

Reply:
Didn't notice that method! Thanks.